data/remnant ships.txt

```diff
@@ -438,7 +446,7 @@ ship "Starling"
 		"fuel capacity" 900
 		"ramscoop" 1
 		"cargo space" 36
-		"outfit space" 312
+		"outfit space" 319
```

Comment:
Ship | Extra outfit space
-----|-----
Starling | 11
Starling (Heavy) | 3
Starling (Hunter) | 1
Starling (Sniper) | 1
Starling (Thrasher) | 6

Seeing as how the lowest extra space is 1, Starlings would only need +6 outfit space, not +7, but again, these ships could use outfits expansions. Starlings, like Gulls, lack cooling and only lose low double digits heat capacity from the installation of an outfits expansion, while the room made by installing an outfits expansion would allow for both a scanner and some cooling.

Reply:
>The lack of cooling I believe is a failure on the part of the design of the original outfit set.

I'd disagree, given that they don't actually need the cooling (and some variants simply can't fit any given their current extra space).

Reply:
Well, be that as it may, the change to their space has been reverted and scanners either fit without the changed space, or use expansions to do so. So this point is resolved.